homeassistant/components/binary_sensor/concord232.py

```diff
@@ -118,7 +118,7 @@ def name(self):
     def is_on(self):
         """Return true if the binary sensor is on."""
         # True means "faulted" or "open" or "abnormal state"
-        return bool(self._zone['state'] == 'Normal')
+        return bool(self._zone['state'] == 'Tripped')
```

Comment:
I think you should check against `'Normal'` according to the comment, since the zone can take multiple states that are not normal:
https://github.com/JasonCarter80/concord232/blob/5ec7987beb5fb6522322936fc52b4f0530b52eea/concord232/concord_commands.py#L129-L136
```py
return bool(self._zone['state'] != 'Normal')
```

Reply:
I updated it to reflect this change.